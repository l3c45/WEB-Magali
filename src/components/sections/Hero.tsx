import { FC } from "react";

const Hero: FC = () => {
  return (
    <div id="home" className="hero__container">
      <div className="hero__image">
        <span>Salon de belleza </span>
        <h1> Magali</h1>
      </div>
    </div>
  );
};

export default Hero;
