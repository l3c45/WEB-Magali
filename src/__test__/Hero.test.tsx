import { render, screen } from '../utils/testUtils';
//import  "../__mock__/interseccionObserber"
import Hero from '../components/sections/Hero';

describe('Testing Hero Component', () => {

beforeEach(()=>{
  render(<Hero />);
})

  it('Should render title ', () => {
    const title="Magali"

    expect(screen.getByText(title)).toMatchSnapshot()
  });
});