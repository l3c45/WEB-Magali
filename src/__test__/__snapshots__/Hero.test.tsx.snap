// Vitest Snapshot v1

exports[`Testing Hero Component > Should render title  1`] = `
<h1>
   Magali
</h1>
`;
